share/OpenMS/CHEMISTRY/Enzymes_RNA.xml

```diff
@@ -55,6 +55,24 @@
       <ITEM name="ThreePrimeGain" value="" type="string" />
       <ITEM name="FivePrimeGain" value="p" type="string" />
     </NODE>
+    <NODE name="mazF">
+      <ITEM name="Name" value="mazF" type="string" />
+      <ITEM name="RegEx" value="(?&lt;=ACA)" type="string" />
+      <ITEM name="RegExDescription" value="mazF cuts before ACA." type="string" />
+      <ITEM name="CutsAfter" value=".*(?!m)$" type="string" />
+      <ITEM name="CutsBefore" value="ACA" type="string" />
+      <ITEM name="ThreePrimeGain" value="p" type="string" />
+      <ITEM name="FivePrimeGain" value="" type="string" />
+    </NODE>
+    <NODE name="colicin_E5">
+      <ITEM name="Name" value="colicin_E5" type="string" />
+      <ITEM name="RegEx" value="(?&lt;=G)U" type="string" />
+      <ITEM name="RegExDescription" value="colicin E5 cuts after G followed by U." type="string" />
```

Comment:
What about the "U" in "CutsAfter"/"CutsBefore"? (Depending on if it cuts "GU" in the middle or after the "U".)

Reply:
According to https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1635277/ it is supposed to cut between the two. It also appears to cut between QU as well. I've pushed a commit that updates this

Reply:
please update Description as well

Reply:
@hendrikweisser I started adding tests, and quickly found that the multi-base specificity doesn't work with the current code. Specificially [this line](https://github.com/OpenMS/OpenMS/blob/55598cf39c73a2092365fbdb51e63defa9b29c6f/src/openms/source/CHEMISTRY/RNaseDigestion.cpp#L106) only looks at the nucleotide immediately after (which will never be equal to a multi-nucleotide string. Any suggestions on how to handle these sorts of enzymes?